packages/data/ace/src/Data_XSSNeutronDataExtractor.hpp

```diff
@@ -103,79 +130,190 @@ class XSSNeutronDataExtractor
   Utility::ArrayView<const Energy> extractAverageHeatingNumbersInMeV() const;
 
   //! Extract the NU block form the XSS array
-  Utility::ArrayView<const double> extractNUBlock() const;
+  inline Utility::ArrayView<const double> extractNUBlock() const
+  {
+    return extractBlock(nu);
+  }
```

Comment:
It's common to write simple inline functions like this on a single line for compactness, especially since there are so many.  
```suggestion
  inline Utility::ArrayView<const double> extractNUBlock() const  { return extractBlock(nu); };
```

Reply:
can do